stl/inc/ranges

```diff
@@ -8402,6 +8402,8 @@ namespace ranges {
     _EXPORT_STD template <class _Container, input_range _Rng, class... _Types>
         requires (!view<_Container>)
     _NODISCARD constexpr _Container to(_Rng&& _Range, _Types&&... _Args) {
+        static_assert(!is_const_v<_Container>, "the target type must be cv-unqualified");
+        static_assert(!is_volatile_v<_Container>, "the target type must be cv-unqualified");

```

Comment:
```suggestion
        static_assert(!is_const_v<_Container>, "the target type must not be const. (N4928 26.5.7.2 [range.utility.conv.to])");
        static_assert(!is_volatile_v<_Container>, "the target type must not be volatile. (N4928 26.5.7.2 [range.utility.conv.to])");
```

Reply:
Ditto ":puke: subclause numbers, just use the stable name"